app/models/katello/glue/provider.rb

```diff
@@ -97,10 +97,7 @@ def owner_upstream_export(upstream, zip_file_path, _options)
         data = Resources::Candlepin::UpstreamConsumer.export("#{url}#{upstream['uuid']}/export", upstream['idCert']['cert'],
                                                              upstream['idCert']['key'], ca_file)
 
-        File.open(zip_file_path, 'w') do |f|
-          f.binmode
-          f.write data
-        end
+        File.write(zip_file_path, data, mode: 'wb')
```

Comment:
I think opening with mode `wb` is the same as opening with mode `w` and then calling `binmode`, but please consider this carefully.

Reply:
According to https://ruby-doc.org/core-2.5.0/IO.html#method-i-binmode they appear to be the same.